src/schema/schema.ts

```diff
@@ -117,8 +117,14 @@ export function createZimbraSchema(
 				getAvailableLocales: (_: any) => client.getAvailableLocales(),
 				getMailboxMetadata: (_: any, variables) =>
 					client.getMailboxMetadata(variables as GetMailboxMetadataOptions),
-				getMessage: (_, variables) =>
-					client.getMessage(variables as GetMessageOptions),
+				getMessage: (_, variables, context = {}) => {
```

Comment:
this code gets executed for rendering the mail detail panel, it's here as we have completed mail detail panel as well with hard coded data in this story